### PR TITLE
Fixed Home Activity fragment transactions 

### LINKED ISSUE #1906: When exiting the application, the application crashes
**Summary:** 

When exiting the application, the application crashes.

**Steps to reproduce:** 

https://user-images.githubusercontent.com/94394661/221494436-eda35fb8-9649-4a67-a0c8-6780eb88a869.mp4

**Device and Android version:** 

Device : OnePlus Nord
Android 12

### PR DESCRIPTION
**Pull request by Aditya-gupta99**
Fixes #1906 

https://user-images.githubusercontent.com/94394661/222082816-3b13608b-04cc-4929-a549-74bf34b8ec32.mp4

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Aditya-gupta99** [comment]
@avinash14022002 Now the PR has been separated, please review it.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/HomeActivity.kt [hunk 1]
```diff
@@ -60,9 +60,7 @@ open class HomeActivity : MifosBaseActivity(), NavigationView.OnNavigationItemSe
 
     private var menu: Menu? = null
 
-    private var itemClient = true
-    private var itemCenter = true
-    private var itemGroup = true
+    private var mCount = 1
```
Comment (avinash14022002) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/HomeActivity.kt:63]: @Aditya-gupta99 ,
please use proper naming convention

**Review by avinash14022002 — CHANGES_REQUESTED** [review]
Do the changes

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/HomeActivity.kt [hunk 2]
```diff
@@ -229,16 +207,13 @@ open class HomeActivity : MifosBaseActivity(), NavigationView.OnNavigationItemSe
...
-                super.onBackPressed()
-                return
+            if(!onSearchFragment) {
+            goHomeFragment()
             }
-            doubleBackToExitPressedOnce = true
-            Toast.makeText(this, R.string.back_again, Toast.LENGTH_SHORT).show()
-            Handler().postDelayed({ doubleBackToExitPressedOnce = false }, 2000)
+            else{
+            doubleBackToExit()
```
Comment (avinash14022002) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/HomeActivity.kt:214]: @Aditya-gupta99 ,
please check why lint and Pretier didn't detect that?

**Review by avinash14022002 — CHANGES_REQUESTED** [review]
please do the needful changes

**Comment by Aditya-gupta99** [comment]
We're closing this PR since it's already fixed

**Pull request closed without merging**